src/Mapping/ClassMetadata.php

```diff
@@ -482,98 +482,80 @@
     private const ALLOWED_GRIDFS_FIELDS = ['_id', 'chunkSize', 'filename', 'length', 'metadata', 'uploadDate'];
 
     /**
-     * READ-ONLY: The name of the mongo database the document is mapped to.
-     *
-     * @var string|null
+     * The name of the mongo database the document is mapped to.
      */
-    public $db;
+    public private(set) ?string $db = null;
```

Comment:
`public` is the default. I think our CS should be to not set it?
```suggestion
   private(set) ?string $db = null;
```

@doctrine/maintainers WDYT?

Reply:
This version is perfect.